pages/en/lb2/Model-definition-JSON-file.md

```diff
@@ -101,6 +101,13 @@ Properties are required unless otherwise designated.
       </td>
     </tr>
     <tr>
+      <td>forceId</td>
+      <td>Boolean</td>
+      <td>
+        The default value is set to true whenever a model uses an auto-generated ID. If it is set to true it prevents users from setting the auto-id value manually.
```

Comment:
Thanks @bajtos ! I'm happy that I mentioned you in the PR to catch that https://github.com/strongloop/loopback-datasource-juggler/pull/982 is just made against LB3.x and not back-ported.

@crandmck I made https://github.com/strongloop/loopback.io/pull/178 to rectify default value of `forceId` for LB2.x; in terms of LB3.x, I do not see any pages `pages/en/lb3/Model-definition-JSON-file.md`, what is your plan in this regard? Can we copy & paste `pages/en/lb2/Model-definition-JSON-file.md` to `pages/en/lb3/Model-definition-JSON-file.md`; I believe before copying&pasting the page to under `pages/en/lb3/` directory the page needs some modification?